integration/testnet/network.go

```diff
@@ -470,122 +471,6 @@ func (n *NetworkConfig) Swap(i, j int) {
 	n.Nodes[i], n.Nodes[j] = n.Nodes[j], n.Nodes[i]
 }
 
-// NodeConfig defines the input config for a particular node, specified prior
-// to network creation.
-type NodeConfig struct {
```

Comment:
can these big refactorings be summarized in the PR summary? Where is all this code moved to?

Reply:
This file was to long, moved NodeConfig and funcs to node_config.go https://github.com/onflow/flow-go/pull/3877/files#diff-e4bfa966385ecca494a7ceecd12d8182f3a6d1dbd2a819561620c0ecc589f966